dev-util/bitrise/bitrise-1.24.0.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit golang-build
+
+EGO_ON="github.com/bitrise-io"
+EGO_PN="${EGO_ON}/${PN}"
+
+DESCRIPTION="Run your Bitrise.io automations on any Mac or Linux machine"
+HOMEPAGE="https://www.bitrise.io/cli"
+SRC_URI="https://${EGO_PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="doc"
+
+RDEPEND=">=dev-util/envman-2.1.1
+	>=dev-util/stepman-0.10.5"
+
+S="${WORKDIR}/src/${EGO_ON}/${PN}"
+
+src_unpack() {
+	default
+	mkdir -p "${WORKDIR}/src/${EGO_ON}" || die "Couldn't create project dir in GOPATH"
+	mv "${WORKDIR}/${P}" "${WORKDIR}/src/${EGO_ON}/bitrise" || die "Couldn't move sources to GOPATH"
+}
+
+src_compile() {
+	GOPATH="${WORKDIR}" go build -v -o bin/bitrise || die "Couldn't compile bitrise"
+}
+
+src_test() {
+	pushd _tests/integration > /dev/null || die "Couldn't find integration tests directory"
+	rm envstore_test.go exit_code_test.go global_flag_test.go json_params_test.go log_filter_test.go output_alias_test.go \
+		step_template_test.go trigger_params_test.go update_test.go timeout_test.go || die "Couldn't remove network-dependent tests"
+	popd > /dev/null || die "Couldn't return to ${S} directory"
+	local -x PULL_REQUEST_ID=""
```

Comment:
local variables are usually written with smallcase. Are these being used globally/locally by golang-build.eclass, because they don't seem to be used in this ebuild? Same applies to line below.

Reply:
It was a request from previous review to make it local variable: https://github.com/gentoo/gentoo/pull/10137#discussion_r224988036
They are not used by golang eclass.